package.json

```diff
@@ -22,13 +22,13 @@
   "prettier": {},
   "dependencies": {
     "@octokit/request-error": "^2.0.2",
+    "@octokit/webhooks-definitions": "3.57.2",
```

Comment:
So this is a tough one - I think if we're going to make this a dependency (peer or otherwise) we should loosen our constraints to enable deduplication and getting fixes etc.

This will technically open us up for becoming out of sync, but the chances and implications of that are very minor: the only changes in the library that impacts us here is if we add/remove/change an event (as in, event name), which can be worked around as outlined in #434 (i.e, with casting, and because we're not erroring on unknown events, the user would just get a warning).